.github/workflows/deploy.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           SLACK_COLOR: '#3278BD'
           SLACK_ICON: https://covid.gov.cz/favicon.svg
           SLACK_USERNAME: "Nový build"
-          SLACK_MESSAGE: "Mame nový build @Honza Vlasák :tada:"
+          SLACK_MESSAGE: "Mame nový build @U01CY3E4WMD :tada:"
```

Comment:
@miromichalicka try this: `SLACK_MESSAGE: "Mame nový build <@U01CY3E4WMD> :tada:"`